doc/cli.markdown

```diff
@@ -806,7 +806,7 @@ select a service variable (may be used together with the --device option)
 
 do not prompt for confirmation before deleting the variable
 
-## env add NAME [VALUE]
+## env add &#60;name&#62; [value]
```

Comment:
Also if we were to make this change to `env add` and other previously converted commands, we'd have to change other help sections as well, like ARGUMENTS and EXAMPLES below:

**Before this PR:** (usage line consistent with ARGUMENTS and EXAMPLES sections)
```
$ balena help env add
Add an environment or config variable to an application, device or service.

USAGE
  $ balena env add NAME [VALUE]

ARGUMENTS
  NAME   environment or config variable name
  VALUE  variable value; if omitted, use value from this process' environment

EXAMPLES
  $ balena env add TERM --application MyApp
  $ balena env add EDITOR vim --application MyApp
  $ balena env add EDITOR vim --application MyApp --service MyService
  $ balena env add EDITOR vim --device 7cf02a6
  $ balena env add EDITOR vim --device 7cf02a6 --service MyService
```
**After this PR:** (usage line inconsistent with ARGUMENTS and EXAMPLES sections)
```
$ ./bin/balena help env add
Add an environment or config variable to an application, device or service.

USAGE
  $ balena env add <name> [value]

ARGUMENTS
  NAME   environment or config variable name
  VALUE  variable value; if omitted, use value from this process' environment

EXAMPLES
  $ balena env add TERM --application MyApp
  $ balena env add EDITOR vim --application MyApp
  $ balena env add EDITOR vim --application MyApp --service MyService
  $ balena env add EDITOR vim --device 7cf02a6
  $ balena env add EDITOR vim --device 7cf02a6 --service MyService
```

Reply:
Yes, I think this needs to be part of the wider discussion.